### PR TITLE
Update disk_encryption.py

### PR DESCRIPTION
**Pull request by ofields**
Fixed misleading error message

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [ ] Each command and parameter has a meaningful description.
- [ ] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by troydai** [comment]
Thank you @ofields

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/disk_encryption.py [hunk 1]
```diff
@@ -72,8 +72,8 @@ def encrypt_vm(resource_group_name, vm_name,  # pylint: disable=too-many-locals,
     # 1. First validate arguments
 
     if not aad_client_cert_thumbprint and not aad_client_secret:
-        raise CLIError('Please provide either --aad-client-id or --aad-client-cert-thumbprint')
-
+        raise CLIError('Please provide either --aad-client-cert-thumbprint or --aad-client-secret ')
+
```
Comment (troydai) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/disk_encryption.py]: I have a feeling the trailing whitespace is going to fail the CI.

**Comment by derekbekoe** [comment]
@ofields Thanks for your contribution. Can you take a look at the failing CI?

**Comment by tjprescott** [comment]
Due to recent changes to the CLI core, you will need to rebase your changes against the latest in the dev branch.

**Comment by troydai** [comment]
@yugangw-msft if this change makes sense, let's correct it and merge it by ourselves.

**Pull request closed without merging**